pyEpiabm/docs/source/conf.py

```diff
@@ -50,12 +50,15 @@
     autodoc_default_options = {
         'members': None,
         'inherited-members': None,
+        'imported-memebers': None,
```

Comment:
```suggestion
        'imported-members': None,
```
It seemed to work anyway but I assume this is a typo?

Reply:
Good catch, @KCGallagher! I forgot to take that out, we don't actually need it